tagstudio/src/qt/modals/tag_database.py

```diff
@@ -77,13 +101,18 @@ def update_tags(self, query: str | None = None):
             row = QHBoxLayout(container)
             row.setContentsMargins(0, 0, 0, 0)
             row.setSpacing(3)
-            tag_widget = TagWidget(tag, has_edit=True, has_remove=False)
+            tag_widget = TagWidget(tag, has_edit=True, has_remove=True)
             tag_widget.on_edit.connect(lambda checked=False, t=tag: self.edit_tag(t))
+            tag_widget.on_remove.connect(lambda t=tag: self.remove_tag(t))
```

Comment:
I believe, the codebase uses lambda throughout when using slots. Is there a real world benefit to using a partial here (or throughout)?

Reply:
in this case, partial feels comfortable for me because that makes the intent more explicit and ties directly to the idea of binding `tag` to `self.remove_tag`. however i would say it's a matter of style so i'm okay if @DandyDev01 decides to go with lambda here.

Reply:
Because I haven't seen partials used too much elsewhere in the code base, I'll just stick with the lambda in this case.